scripts/create_x509.go

```diff
@@ -32,7 +32,7 @@ func generateCert() error {
 		basePath += "/"
 	}
 
-	if err := os.MkdirAll(basePath, os.ModePerm); err != nil {
+	if err := os.MkdirAll(basePath, 0750); err != nil {
```

Comment:
why do we prefer the number to the constant now? what was the linter complaint?

Reply:
test/helper/e2e/utils/utils.go:40:9: G301: Expect directory permissions to be 0750 or less (gosec)
        err := os.MkdirAll(filepath.Dir(path), os.ModePerm)

Reply:
I think it can be replaced with `fs.ModeDir`